dev-java/cdegroot-db/cdegroot-db-0.08-r3.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+JAVA_PKG_IUSE="doc source"
+
+inherit java-pkg-2 java-ant-2
+
+DESCRIPTION="OO database written in Java"
+HOMEPAGE="http://cdegroot.com/software/db/"
+SRC_URI="http://cdegroot.com/software/db/download/com.${P/-/.}.tar.gz"
+
+LICENSE="MIT"
+SLOT="1"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+IUSE="doc source"
+
+DEPEND=">=virtual/jdk-1.8:*"
+RDEPEND=">=virtual/jre-1.8:*"
```

Comment:
```suggestion
DEPEND=">=virtual/jdk-1.8"
RDEPEND=">=virtual/jre-1.8"
```

Reply:
@monsieurp without `:*` there's a warning (or at least it was) that the slot is not specified though it should, so we stick with this.

Reply:
@fordfrog 
https://bugs.gentoo.org/785532